activemodel/lib/active_model/attribute_methods.rb

```diff
@@ -18,10 +18,10 @@ class MissingAttributeError < NoMethodError
   # = Active \Model \Attribute \Methods
   #
   # Provides a way to add prefixes and suffixes to your methods as
-  # well as handling the creation of <tt>ActiveRecord::Base</tt>-like
+  # well as handling the creation of +ActiveRecord::Base+-like
```

Comment:
```suggestion
  # well as handling the creation of ActiveRecord::Base - like
```
(not sure about this one, are extra spaces needed to make sure it links?)

Reply:
There are lots of places where "returns an array of ActiveRecord::Base" and I wonder if there is a better way 🤔